manifests/addons/values-kiali.yaml

```diff
@@ -1,13 +1,10 @@
-# Create a Kiali CR in the istio-system namespace, which is where Kiali will be installed.
 # We set up anonymous authentication as this is for demos.
-cr:
-  create: true
-  namespace: istio-system
-  spec:
-    auth:
-      strategy: anonymous
-    deployment:
-      accessible_namespaces:
-      - '**'
-      image_version: v1.22
-      ingress_enabled: false
+auth:
+  strategy: anonymous
+deployment:
+  accessible_namespaces:
+  - '**'
+  image_version: v1.22
+  ingress_enabled: false
+login_token:
+  signing_key: CHANGEME
```

Comment:
Just curious, if this is unset is the behavior any different?

Reply:
If you leave this unset, a random string will be generated for you by the helm chart. But because we are running helm template and generating an all-in-one, the all-in-one samples/addons/kiali.yaml will simply have a random string in this setting and everyone will be basically using the same random string. It is not secure since it will be a well-known random string that everyone is using, but it may "look" secure to some people and might confuse them.

Not that it matters - the default auth strategy is anonymous so anyone can get in anyway.  But I thought it better to change this to something more obvious.

Reply:
Ah right, that makes sense. Thanks